apps/mail/hooks/use-categories.ts

```
@@ -15,29 +13,34 @@ export interface CategorySetting {
+          id: 'Inbox',
+          name: 'Inbox',
+          searchValue: 'in:inbox',
+          order: 1,
+          isDefault: false
+        }
+      ];
+    }
+    
     return sorted;
-  }, [data?.settings.categories, defaultCategories]);
+    return sorted;
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)